packages/amplify-codegen/src/utils/getRelativeTypesPath.js

```diff
@@ -5,7 +5,7 @@ function getRelativeTypesPath(opsGenDirectory, generatedFileName) {
     const relativePath = path
       .relative(opsGenDirectory, generatedFileName)
       // ensure posix path separators are used
-      .split(path.win32.sep)
+      .split(path.win32?.sep || '\\')
```

Comment:
Why do we fallback to `\`? Does this code get executed only in windows?

Reply:
This is executed in all environments. `path.win32` is not implemented by `path-browserify`. When in browser `\` will be used, when in node `path.win32.sep` (which is `\`) will be used.